raiden-dapp/src/router/index.ts

```diff
@@ -70,6 +70,28 @@ const router = new Router({
         title: 'Channels'
       },
       component: () => import('../views/Channels.vue')
+    },
+    {
+      path: '/general',
+      name: RouteNames.GENERAL,
+      beforeEnter: (to, from, next) => {
+        if (to.matched.length) {
+          to.matched[0].components.default = from.matched[0].components.default;
+          to.matched[0].components.modal = () =>
+            import('../views/GeneralDialog.vue');
```

Comment:
I don't understand this part. Does it only import `GeneralDialog` when the `/general` route is accessed?
What is in `.default`?

Reply:
Has been clarified via DM